### PR TITLE
Printing levels

### PR DESCRIPTION
**Pull request by SkySkimmer**
On top of #260.

### CODE REVIEW

**Comment by SkySkimmer** [comment]
close #262

File: src/utils/level.ml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+type t = int
```
Comment (andrejbauer) [inline src/utils/level.ml]: Could we get away with `t` being abstract? Maybe if we expose a comparison function for levels?

Comment (SkySkimmer) [inline src/utils/level.ml]: It is abstract, you're looking at the .ml

Comment (SkySkimmer) [inline src/utils/level.ml]: It may be a good idea to expose the comparison instead of using `(<) : 'a -> 'a -> bool` though.

**Pull request merged**